oktoberfest/data/spectra.py

```diff
@@ -229,7 +276,23 @@ def from_csv(cls: Type[SpectraT], input_file: Union[str, Path]) -> SpectraT:
         :return: a spectra instance
         """
         input_file = str(input_file)
-        spectra = cls()
-        spectra.spectra_data = pd.read_csv(input_file)
+        all_columns = pd.read_csv(input_file)
+        size = (all_columns.shape[0], 174)
+        spectra = cls(size)
+        spectra.add_columns(all_columns)
```

Comment:
I think generally, this function should be completey removed and instead only anndata functions should be directly used, i..e. generating layers and X directly. Leave it like that for now and deal with the other things, I'll have a look afterwards then

Reply:
As far as I can tell it is only used in the nox test runs.